nodewatcher/plugins/slurm.py

```diff
@@ -75,3 +86,10 @@ def lockHost(hostname, unlock=False):
         run_command(command)
     except subprocess.CalledProcessError:
         log.error("Error %s host %s", "unlocking" if unlock else "locking", hostname)
+
+
+def _get_node_slots():
+    hostname = check_command_output("hostname")
+    command = "/opt/slurm/bin/sinfo -o '%c' -n {0} -h".format(hostname)
```

Comment:
Can you put a comment with example output?

Reply:
Sure! Will do in a separate commit as part of the next PR in order to close this one